pkg/util/cloudfoundry/types_test.go

```diff
@@ -238,3 +279,39 @@ func TestGetVcapServicesMap(t *testing.T) {
 	assert.Nil(t, err)
 	assert.EqualValues(t, expected, result)
 }
+
+func TestIsAllowedTag(t *testing.T) {
+	// when both empty, exclude everything
+	includeList := []*regexp.Regexp{}
+	excludeList := []*regexp.Regexp{}
+
+	result := isAllowedTag("aRandomValue", includeList, excludeList)
+	assert.EqualValues(t, false, result)
+
+	// include strings in the includeList and not in the excludeList
+	includeList = []*regexp.Regexp{regexp.MustCompile("include.*")}
+	excludeList = []*regexp.Regexp{}
+
+	result = isAllowedTag("includeTag", includeList, excludeList)
+	assert.EqualValues(t, true, result)
+
+	result = isAllowedTag("excludeTag", includeList, excludeList)
+	assert.EqualValues(t, false, result)
+
+	// reject strings in the excludeList
+	includeList = []*regexp.Regexp{}
+	excludeList = []*regexp.Regexp{regexp.MustCompile("exclude.*")}
+
+	result = isAllowedTag("aRandomValue", includeList, excludeList)
+	assert.EqualValues(t, true, result)
+
+	result = isAllowedTag("excludeTag", includeList, excludeList)
+	assert.EqualValues(t, false, result)
+
+	// reject strings in the excludeList even if they exist in the includeList
+	includeList = []*regexp.Regexp{regexp.MustCompile("include.*")}
+	excludeList = []*regexp.Regexp{regexp.MustCompile("exclude.*"), regexp.MustCompile("include.*")}
+
+	result = isAllowedTag("includeTag", includeList, excludeList)
+	assert.EqualValues(t, false, result)
```

Comment:
Just to be more precise
```suggestion
	includeList = []*regexp.Regexp{regexp.MustCompile("include.*")}
	excludeList = []*regexp.Regexp{regexp.MustCompile("includeExclude.*")}

	result = isAllowedTag("includeTag", includeList, excludeList)
	assert.EqualValues(t, true, result)
	result = isAllowedTag("includeExcludeTag", includeList, excludeList)
	assert.EqualValues(t, false, result)
```